### DISCUSSION TITLE
Using kitty for nnn's detached text feature

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [kovidgoyal/kitty#4024] [Q&A] [answered]
I recently switched to nnn and in its wiki, there's an example to open a text file in a [xfce-terminal window](https://github.com/jarun/nnn/wiki/Basic-use-cases#detached-text). I would like to achieve something similar but instead use kitty. 

I know this is possible with kitty's remote control protocol and figured out that I'd need `kitty @launch nvim` but what I'm unable to figure out is how to pass nnn's present selection to this instance of kitty. The example uses `\"$*\"` but this doesn't work for when running `kitty @launch nvim "\$"\`. What this does is launch neovim in a new kitty window with the correct filename but it's not the desired file.

What's the right way of doing this?

**Comment by kovidgoyal** [comment, marked as answer]
Not sure what your question is. Running

kitty @ launch vim "/path/to/some/file"

in a shell works fine. If its not working in nnn
run a shell script instead that logs the argument passed to it, to check
what nnn is passing to it.
